training/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,29 @@
+Cloud Services and Infrastructure in Digital Publishing
+===========================
+
+We use Amazon Web Services in Digital Publishing to host all of our services, as well as our Continuous Integration (CI).
```

Comment:
Minor thing, but it might be worth putting `(AWS)` after Amazon Web Services for anyone who doesn't know or tries searching for AWS.